locustfile.py

```diff
@@ -60,3 +63,16 @@ def idemia_enrollment(self):
             headers={"Authorization": self.bearer_token},
             name="/idemia/enrollment/uuid",
         )
+
+
+class SKTestUser(HttpUser):
+    wait_time = constant_pacing(1)
+
+    @task(1)
+    def hello_world(self):
+        self.client.post(
+            f"https://{SK_API_ROUTE}/v1/company/wdK3fH48XuoXzvZyeNJEYFA9i8K72BZg/flows/IU1iDIvviIth5jiYmNvgsS43Kg29RxyB/start",
+            data={},
+            auth=None,
```

Comment:
`auth` is an optional keyword argument. Setting it to none is essentially a no-op.
```suggestion
```